src/server/pfs/server/driver.go

```diff
@@ -1708,13 +1722,15 @@ func (d *driver) deleteFile(ctx context.Context, file *pfs.File) error {
 }
 
 func (d *driver) deleteAll(ctx context.Context) error {
-	repoInfos, err := d.listRepo(ctx, nil)
+	repoInfos, err := d.listRepo(ctx, nil, true)
 	if err != nil {
 		return err
 	}
-	for _, repoInfo := range repoInfos {
-		if err := d.deleteRepo(ctx, repoInfo.Repo, true); err != nil {
-			return err
+	for i, repoInfo := range repoInfos.RepoInfo {
+		if repoInfos.Scopes[i] >= auth.Scope_WRITER {
```

Comment:
I'm not sure if we need this check--deleteRepo already checks if the caller is an owner (if we can get rid of it, we can also set includeAuth to false above)